src/replica/duplication/load_from_private_log.cpp

```diff
@@ -146,6 +146,8 @@ void load_from_private_log::run()
 
 void load_from_private_log::find_log_file_to_start()
 {
+    _duplicator->set_duplication_plog_checking(true);
```

Comment:
Would it be fine to return from the middle of the function without roll back by `_duplicator->set_duplication_plog_checking(false);` ?